### DISCUSSION TITLE
Bootstrap doesn't allow more than one instance per element

### DISCUSSION THREAD

**Discussion by MohammedMubeen** [twbs/bootstrap#38047] [Q&A]
Hi Folks,

Please help I facing following error same web page.

Bootstrap doesn't allow more than one instance per element. Bound instance: bs.popover.
Bootstrap doesn't allow more than one instance per element. Bound instance: bs.tooltip.

**Comment by julien-deramond** [comment]
Hey @MohammedMubeen 
Without providing any source code, it's not possible to help you. Maybe you could try to share something with [CodePen](https://codepen.io) (or any other tool).
Maybe there's an answer for you in previous closed issues; like this one https://github.com/twbs/bootstrap/issues/37749#issuecomment-1367596448

**Reply by MohammedMubeen** [reply]
Hi @julien-deramond ,

Thanks for the response.

I have attached screen shot for your reference.

Button hover tooltip & popover overlap, Please help me how to fix it in BS 5.

<i class="fa-light fa-trash-can text-danger pointer" data-bs-toggle="tooltip" data-bs-placement="left" data-bs-original-title="Delete this quotation" aria-hidden="true"></i>


![image](https://user-images.githubusercontent.com/26380140/218657633-de9cc11e-b37a-4025-9341-780dab1086af.png)
![image](https://user-images.githubusercontent.com/26380140/218657735-77baca73-24ca-4931-9886-acd0dd400fbc.png)
![image](https://user-images.githubusercontent.com/26380140/218657989-b5c5b9b0-225c-426e-9883-58534c6f51eb.png)

**Reply by MohammedMubeen** [reply]
Hi @julien-deramond ,

I found that issue will happen, While open modal , tooltip instance will create more than 1000 times.

```
<span id="lnkDeleteQuotation" data-ng-click="Delete_Click()"><i class="fa-light fa-trash-can text-danger pointer" data-bs-toggle="tooltip" data-bs-placement="left" data-bs-original-title="Delete this quotation" aria-hidden="true" ></i></span>

$scope.Delete_Click= function (QuotItem) {
        $('#QuotDeleteWarningPopup').modal('show');
    };


<div id="QuotDeleteWarningPopup" class="bootbox modal delete_modal_center fade" tabindex="-1" role="dialog" aria-labelledby="myModalLabel1" aria-hidden="true" data-bs-keyboard="false" data-bs-backdrop="static">
            <div class="modal-dialog" style="width: 100%">
                <div class="modal-content">
                    <div class="modal-header bg-danger text-white delete_modal_header">
                        <h4 class="modal-title">Confirm Deletion</h4>
                        <i class="fa-light fa-times closeicon text-white mt-1 delete_modal_close" aria-hidden="true" data-bs-dismiss="modal" aria-label="Close"></i>
                    </div>
                    <div class="modal-body delete_modal_body">
                        <div class="bootbox-body text-center">
                            <h6>Are you sure you want to delete this quotation?</h6>
                        </div>
                    </div>
                    <div class="modal-footer">
                        <button type="button" id="LinkButton4" runat="server" data-bs-dismiss="modal" class="btn btn-outline-secondary btn-md">Cancel</button>
                        <button type="button" id="btnDeleteQuot" runat="server" class="btn btn-danger btn-md" data-ng-click="ConfirmDeleteQuot_Click()">Delete</button>
                    </div>
                </div>
            </div>
        </div>
```
![image](https://user-images.githubusercontent.com/26380140/218925733-5f1a94f4-5f24-4ed4-b532-481fa853b673.png)
